tests/unit/ui/components/TestRating.js

```diff
@@ -142,9 +142,20 @@ describe(__filename, () => {
     verifyRating(root);
   });
 
+  it('renders an appropriate title when updating a rating', () => {
+    const root = render({ rating: 3 });
+
+    [1, 2, 3].forEach((rating) => {
+      expect(root.ratingElements[rating].title).toEqual(`Update your rating to ${rating} out of 5.`);
+    });
+    [4, 5].forEach((rating) => {
+      expect(root.ratingElements[rating].title).toEqual(`Update your rating to ${rating} out of 5.`);
```

Comment:
I don't understand why there are two arrays here.

Reply:
That is a holderover from the old test I think, but if the rating is 3 this should be updated to have "Update your rating to X" for everything except 3, and then a "Your rating is currently 3/5" or even just no title for 3...

Reply:
@tofumatt is right - here's the old test for reference: https://github.com/mozilla/addons-frontend/blob/master/tests/unit/ui/components/TestRating.js#L195